base/sg_cluster_cfg.go

```diff
@@ -51,19 +53,20 @@ func NewCfgSG(bucket Bucket) (*CfgSG, error) {
 		bucket:        bucket,
 		loggingCtx:    loggingCtx,
 		subscriptions: make(map[string][]chan<- cbgt.CfgEvent),
+		groupID:       groupID,
```

Comment:
As in the previous case, would prefer to define the computed prefix here instead of groupID, unless there's another reason to store groupID.  In this case we're paying for the concatenation with cfgKey anyway, but it's still better to make it clear what groupID is needed for.